src/actions/ast.js

```diff
@@ -59,6 +60,10 @@ export function setSymbols(sourceId: SourceId) {
       [PROMISE]: getSymbols(source.id)
     });
 
+    if (isPaused(getState())) {
+      await dispatch(mapFrames());
```

Comment:
There's a little bit of unnecessary work going on here. Not sure if it's enough to matter.

Reply:
I think that's fine. looks good to me!